A/ASL/build_tarballs.jl

```diff
@@ -0,0 +1,92 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "ASLBuilder"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "http://netlib.org/ampl/solvers.tgz" =>
```

Comment:
Isn't there a versioned tarball?  This is not very good for reproducibility.  Where does the v0.1.0 above come from?

Reply:
It's v1 of the builder. The source isn't versioned. I know it's not good. The author just won't have it.